ada/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+
+
+class RedirectUrl(models.Model):
+
+    url = models.URLField(
+        max_length=255,
+        blank=True,
+        null=True,
+        default="https://hub.momconnect.za/confirmredirect",
+    )
+    content = models.TextField(default="This entry has no copy")
+    refresh_url = models.CharField(
+        max_length=200, null=False, blank=False, default="https://hub.momconnect.za"
+    )
```

Comment:
For this, we're redirecting back to ourselves, so we don't need a database field for this